lib/incito-browser/incito.js

```diff
@@ -292,7 +291,7 @@ export default class Incito extends MicroEvent {
             attrs.muted = '';
             attrs.playsinline = '';
             attrs.preload = 'none';
-            attrs.poster = 'noposter';
+            attrs.poster = view.src + '#t=0.1';
```

Comment:
I don't think this will work as expected, but I didn't test it. Wouldn't this attribute need to point to an image?

Reply:
I don’t think this is the way either.